### DISCUSSION TITLE
Keyboard shortcuts to change Fan/Power Mode + GPU mode + Screen refresh rate simultaneously

### DISCUSSION THREAD

**Discussion by rramnel** [seerge/g-helper#2147] [Ideas]
Currently its possible to change different profiles for the fan/power mode using a keyboard shortcut on the G14. Would it also be possible to have keyboard shortcuts to simultaneously change the fan/power mode + GPU mode + screen refresh rate? 

For e.g. 
Keyboard shortcut 1 - Balanced mode + Standard GPU mode + 120 Hz
Keyboard shortcut 2 - Custom Fan/Power mode + Eco Mode + 60Hz

**Comment by seerge** [comment]
@rramnel can you explain a practical reason for that ?

If you switch between power and battery that would be already automated

**Reply by rramnel** [reply]
Yes I noticed that, but could it also be trigerred via a keyboard shortcut? My use case is that I also game on battery (not connected to the charger) and when doing so would be great to trigger the automation to set the right refresh rate, gpu mode and fan profiles together

**Reply by seerge** [reply]
It can't cause it's a complicated chain of events, that needs to be done in a specific order

**Reply by rramnel** [reply]
Got it, thanks. Really appreciate the great work you are doing with G-Helper!
